clang/test/CodeGenSPIRV/spirv-builtin-lookup-group.cpp

```diff
@@ -0,0 +1,114 @@
+// RUN: %clang_cc1 -cl-std=CL2.0 -fdeclare-spirv-builtins -emit-llvm %s -o - | FileCheck %s
+
+// Note -cl-std=CL2.0 is added to avoid error
+//    Assertion `!isNull() && "Cannot retrieve a NULL type pointer"' failed.
+// Root cause is unknown.
```

Comment:
@Fznamznon @Naghasan @steffenlarsen could you please advise why I'm seeing this error?

Reply:
I think the compiler chocks on `event_t` when building the `__spirv_GroupAsyncCopy ` AST decl, the type would only be initialised internally if sycl or opencl is enabled. Adding `-cl-std=` will force opencl from what I can. A "cleaner" here way would be to have `-x cl` or simply rename the file to `.cl`.

Reply:
thanks @Naghasan 
I've changed file to .cl